packages/aws-cdk-lib/aws-ec2/lib/vpc-endpoint.ts

```diff
@@ -265,6 +266,7 @@ export class InterfaceVpcEndpointAwsService implements IInterfaceVpcEndpointServ
   public static readonly AIRFLOW_OPS = new InterfaceVpcEndpointAwsService('airflow.ops');
   public static readonly APIGATEWAY = new InterfaceVpcEndpointAwsService('execute-api');
   public static readonly APP_MESH = new InterfaceVpcEndpointAwsService('appmesh-envoy-management');
+  public static readonly APP_MESH_OPS = new InterfaceVpcEndpointAwsService('appmesh');
```

Comment:
This one is named weird but don't have much choice.

Reply:
Might be worth deprecating `APP_MESH` and recreating `APP_MESH_ENVOY_MANAGEMENT`, or even `APPMESH_ENVOY_MANAGEMENT`

Reply:
Updated this as well